zyte_common_items/processors.py

```diff
@@ -330,6 +363,45 @@ def aggregateRating(self):
     return value
 
 
+def images_processor(value: Any, page: Any) -> Any:
```

Comment:
it seems page argument is unused - can we remove it?

Reply:
Although I see other processors are written this way as well; it makes sense to keep it for consistency for now, and address later if needed.